intake/cli/client/tests/test_local_integration.py

```diff
@@ -88,20 +91,22 @@ def test_discover():
 
 def test_get_pass():
     cmd = [ex, '-m', 'intake.cli.client', 'get', TEST_CATALOG_YAML, 'entry1']
-    process = subprocess.Popen(cmd, stdout=subprocess.PIPE)
+    process = subprocess.Popen(cmd, stdout=subprocess.PIPE,
+                               universal_newlines=True)
     out, _ = process.communicate()
 
-    assert 'Charlie1   25.0     3' in out.decode('utf-8')
-    assert 'Eve2   25.0     3' in out.decode('utf-8')
+    assert 'Charlie1   25.0     3' in out
```

Comment:
Potential py2/3 problem here?

Reply:
Not sure. We should add py2 to the test matrix.

Reply:
py2 is a separate issue, probably needing a similar amount of work, if we choose to pursue it. I am just noting it for now.